araig_interpreters/src/location_interpreters/vicon_interpreter.cpp

```diff
@@ -0,0 +1,135 @@
+#include <ros/ros.h>
+#include <tf/transform_broadcaster.h>
+#include <tf/transform_listener.h>
+#include <geometry_msgs/TransformStamped.h>
+#include <geometry_msgs/PoseStamped.h>
+#include <std_msgs/String.h>
+#include <ros/master.h>
+#include <vector>
+#include <map>
+#include <string>
+#include <boost/algorithm/string.hpp>
+#include <boost/bind.hpp>
+
+class vicon_interpreter
```

Comment:
Follow ROS Cpp Style Guide for naming classes: [link](http://wiki.ros.org/CppStyleGuide#Classes_.2F_Types)   
i.e. CamelCase for classes and lowerCamelCase for functions